txscript/bench_test.go

```diff
@@ -5,47 +5,40 @@
 package txscript
 
 import (
-	"bytes"
 	"fmt"
 	"io/ioutil"
 	"testing"
 
 	"github.com/decred/dcrd/wire"
 )
 
-var (
-	// manyInputsBenchTx is a transaction that contains a lot of inputs which is
-	// useful for benchmarking signature hash calculation.
-	manyInputsBenchTx wire.MsgTx
```

Comment:
you probably didn't mean to add these changes to this commit

Reply:
oops, removed.